code/_globalvars/lists/xenobiology.dm

```diff
@@ -2,10 +2,33 @@
 
 ///Assoc list of cell line define | assoc list of datum | cell_line
 GLOBAL_LIST_INIT_TYPED(cell_line_tables, /list, list(
-	CELL_LINE_TABLE_SLUDGE = list(/datum/micro_organism/cell_line/sholean_grapes = 2, /datum/micro_organism/cell_line/blobbernaut = 1, /datum/micro_organism/cell_line/slime = 2, /datum/micro_organism/cell_line/cow = 1, /datum/micro_organism/cell_line/mouse = 1, /datum/micro_organism/cell_line/chicken = 1, /datum/micro_organism/cell_line/cockroach = 2),
-	CELL_LINE_TABLE_MOIST = list(/datum/micro_organism/cell_line/gelatinous_cube = 2, /datum/micro_organism/cell_line/megacarp = 1, /datum/micro_organism/cell_line/slime = 2, /datum/micro_organism/cell_line/snake = 1, /datum/micro_organism/cell_line/carp = 1, /datum/micro_organism/cell_line/cockroach = 1),
-	CELL_LINE_TABLE_BLOB = list(/datum/micro_organism/cell_line/blobbernaut = 1, /datum/micro_organism/cell_line/blob_spore = 1),
-	CELL_LINE_TABLE_MOLD = list(/datum/micro_organism/cell_line/vat_beast = 2, /datum/micro_organism/cell_line/bear = 1, /datum/micro_organism/cell_line/slime = 1, /datum/micro_organism/cell_line/blob_spore = 1, /datum/micro_organism/cell_line/mouse = 2, /datum/micro_organism/cell_line/corgi = 1, /datum/micro_organism/cell_line/cockroach = 1, /datum/micro_organism/cell_line/cat = 1),
+	CELL_LINE_TABLE_SLUDGE = list(/datum/micro_organism/cell_line/sholean_grapes = 2,
+								/datum/micro_organism/cell_line/blobbernaut = 1,
+								/datum/micro_organism/cell_line/slime = 2,
+								/datum/micro_organism/cell_line/cow = 1,
+								/datum/micro_organism/cell_line/mouse = 1,
+								/datum/micro_organism/cell_line/chicken = 1,
+								/datum/micro_organism/cell_line/cockroach = 2),
```

Comment:
This is my only complaint for your whole PR (since it all just seems to be data):

https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#tabs-not-spaces

> Do not use tabs/spaces for indentation in the middle of a code line. Not only is this inconsistent because the size of a tab is undefined, but it means that, should the line you're aligning to change size at all, we have to adjust a ton of other code. Plus, it often time hurts readability.

The list should only be indented up one layer. Apply this suggestion to the rest of your code:

```suggestion
	CELL_LINE_TABLE_SLUDGE = list(
		/datum/micro_organism/cell_line/sholean_grapes = 2,
		/datum/micro_organism/cell_line/blobbernaut = 1,
		/datum/micro_organism/cell_line/slime = 2,
		/datum/micro_organism/cell_line/cow = 1,
		/datum/micro_organism/cell_line/mouse = 1,
		/datum/micro_organism/cell_line/chicken = 1,
		/datum/micro_organism/cell_line/cockroach = 2,
	),
```